ci/installdeps.sh

```diff
@@ -6,21 +6,15 @@ set -xeuo pipefail
 dn=$(dirname $0)
 . ${dn}/libbuild.sh
 
+if [ -z "${SKIP_INSTALLDEPS:-}" ] && test $(id -u) -eq 0; then
+    dnf -y install jq dnf-plugins-core
```

Comment:
Of course we're duplicating https://github.com/coreos/fedora-coreos-config/blob/1d8cebb0991c174d5607b7e99a5522bc488562f5/ci/buildroot/buildroot-reqs.txt#L35 here.  I hope long term the mock/copr people realize that they can just use podman/Kubernetes.  Right now the "build custom SRPM" flow is totally ad-hoc, but they could just instead allow people to use pre-built containers for things e.g.